config/smarty.config.inc.php

```diff
@@ -88,6 +88,7 @@
 smartyRegisterFunction($smarty, 'function', 'widget', 'smartyWidget');
 smartyRegisterFunction($smarty, 'block', 'widget_block', 'smartyWidgetBlock');
 smartyRegisterFunction($smarty, 'modifier', 'classnames', 'smartyClassnames');
+smartyRegisterFunction($smarty, 'function', 'getUrl', array('Link', 'getUrlSmarty'));
```

Comment:
I think we should use the same convention fro all smarty helpers, so maybe "get_url" instead of "getURL"? or just "url" ?